realsense2_camera/include/constants.h

```diff
@@ -94,6 +94,7 @@ namespace realsense2_camera
     const std::string DEFAULT_UNITE_IMU_METHOD         = "";
     const std::string DEFAULT_FILTERS                  = "";
     const std::string DEFAULT_TOPIC_ODOM_IN            = "";
+    const std::string DEFAULT_SERVICE_SAVE_MAP         = "";
```

Comment:
just creating a default name for the service, following what seems to be the standard for this codebase